### ISSUE TITLE
[ERROR] ULX and F1 errors.

### ISSUE THREAD

**Issue by Trop1cal** [rebel1324/NutScript#150]
![yeetle20](https://user-images.githubusercontent.com/30530009/36064630-ebea70b0-0e5b-11e8-97fb-d11f90cb0b77.jpg)

[ERROR] gamemodes/nutscript/gamemode/core/libs/sh_date.lua:45: attempt to perform arithmetic on a nil value
  1. get - gamemodes/nutscript/gamemode/core/libs/sh_date.lua:45
   2. setup - gamemodes/nutscript/gamemode/core/derma/cl_information.lua:135
    3. Init - gamemodes/nutscript/gamemode/core/derma/cl_menu.lua:71
     4. Create - lua/includes/extensions/client/panel/scriptedpanels.lua:37
      5. unknown - gamemodes/nutscript/gamemode/core/hooks/cl_hooks.lua:600

My attributes and custom character preview aren't showing, plus my custom currency isn't what I set it to [Reichsmarks] rather it's set to USD.

Along with I have a plugin which sets noclip as observer, but with my custom ULX ranks the observer setup doesn't work, and my custom ranks have to use !cloak along with !noclip.

Any help would be wonderful!

**Comment by rebel1324** [comment]
Reorganize your issue with numbers. It's p' hard to follow mate.

**Comment by Sweetnut1018** [comment]
This happens when you set the time before the year, 1970.

**Comment by Sweetnut1018** [comment]
You might have to modify observer plugin at gamemodes/nutscript/plugin.

**Comment by Thadah** [comment]
Something to do with Unixtime probably, we will have to change the date library to support dates before 1970
